www/next.config.js

```diff
@@ -581,6 +569,26 @@ module.exports = withMDX({
         source: '/docs/guides/self-hosting',
         destination: '/docs/guides/hosting/overview',
       },
+      {
+        permanent: false,
+        source: '/humans.txt',
+        destination: `/docs/humans.txt`,
+      },
+      {
+        permanent: false,
+        source: '/lawyers.txt',
+        destination: `/docs/lawyers.txt`,
+      },
+      {
+        permanent: false,
+        source: '/.well-known/security.txt.txt',
+        destination: `/docs/.well-known/security.txt.txt`,
+      },
+      {
+        permanent: false,
+        source: '/oss',
+        destination: `/docs/oss`,
+      },
```

Comment:
Was not sure if we want to make the redirect permanent or not, so left it as `false` for now.